packages/coinstac-pipeline/src/controller.js

```diff
@@ -235,6 +239,10 @@ module.exports = {
           trampoline(() => {
             return queue.length
               ? function _cb(...args) {
+                console.log(controllerState);
+                if (controllerState.stopByUser == 'stop'){
+                  err({message: 'Pipeline run is stopped by user'});
```

Comment:
It's good practice to actually create an error when error'ing out, as often upstream code checks for object type. 
`err(new Error('Pipeline run is stopped by user'))`